models/brats_mri_segmentation/configs/metadata.json

```diff
@@ -1,15 +1,16 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.1.0",
+    "version": "0.1.1",
     "changelog": {
+        "0.1.1": "update for MetaTensor",
         "0.1.0": "complete the model package"
     },
-    "monai_version": "0.9.0",
-    "pytorch_version": "1.12.0",
-    "numpy_version": "1.21.2",
+    "monai_version": "0.9.1",
+    "pytorch_version": "1.13.0a0+340c412",
```

Comment:
here could it be a formal pytorch version? like 1.12.0, thus users (and our ci tests) are able to reproduce the environment.

Reply:
Thanks for your advice! I have just changed it.

Reply:
pytorch does not formally release 1.13.0, and we need to test the torchscript module withe the specified model. I will revert this merged PR, and may need your help to modify it again.